packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -97,6 +112,21 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
             setInputDisable(false);
         }
 
+        if (needs_additional) {
+            if (!document_additional) {
+                errors.document_additional =
+                    localize('Please enter your document number. ') +
+                    getExampleFormat(document_type.additional?.example_format);
```

Comment:
Do you need to localize the example? if yes then you need to pass it as a component to the func.

Reply:
We don't have to localize the example since it is in numeric format